aspose/src/main/java/nl/nn/adapterframework/extensions/aspose/pipe/PdfPipe.java

```diff
@@ -164,6 +173,21 @@ public PipeRunResult doPipe(Message input, PipeLineSession session) throws PipeR
 		}
 	}
 
+	private void populateContext(CisConversionResult result, Message message, int index) {
+		if(StringUtils.isNotEmpty(result.getResultFilePath())){
+			FileMessage document = new FileMessage(new File(result.getResultFilePath()));
+			message.getContext().put(CONVERTED_DOCUMENTS_CONTEXT_KEY+index, document);
```

Comment:
I'm not sure that the message-context is the best place to put these documents? Rather put them into the session?